packages/splits-sdk/package.json

```diff
@@ -67,11 +68,15 @@
     }
   },
   "dependencies": {
+    "@urql/core": "^4.3.0",
     "base-64": "^1.0.0",
     "graphql": "^16.8.1",
-    "graphql-request": "^6.1.0"
+    "lodash": "^4.17.21"
```

Comment:
nit: maybe i'm crazy. i'm trying to keep dependencies here to a minimum. dunno if you made heavy use of lodash, if not might be nice to not include it.

or maybe i'm crazy and this isn't important 🤷

Reply:
yeah, can try to remove it, also not crazy lol, good to keep sdks lightweight

Reply:
looks like the subgraph code uses lodash, so maybe we keep it for now?